docs/developers/stored-requests.md

```diff
@@ -0,0 +1,138 @@
+# Stored Requests
+
+This document gives a technical overview of the Stored Impressions feature.
+
+Docs outlining the motivation and uses will be added sometime in the future.
+
+## Quickstart
+
+Configure your server to read stored requests from the filesystem:
+
+```yaml
+ortb2_config:
+  filesystem: true
+```
+
+Choose an ID to reference your request. Throughout this doc, replace {id} with the ID you've chosen.
+
+Add the file `openrtb2_configs/for_requests/{id}.json` and populate it with some Impression data.
```

Comment:
Is 'Impression' a type in the code, hence capitalized?

Reply:
yes... though now that you mention it, I can also just link to the type in the OpenRTB spec here.